app/src/androidTest/java/com/stevesoltys/seedvault/e2e/SeedvaultLargeTest.kt

```diff
@@ -44,6 +45,11 @@ internal abstract class SeedvaultLargeTest :
         resetApplicationState()
         clearTestBackups()
 
+        runCommand("bmgr enable true")
+        sleep(60_000)
```

Comment:
Fine for now but this could become some sort of logcat grep... maybe possible...

Reply:
yeah would be nice to shorten this (and other) times.